plugins/secureboot-paper/src/main/java/net/civmc/secureboot/paper/SecureBootBootstrap.java

```diff
@@ -0,0 +1,19 @@
+package net.civmc.secureboot.paper;
+
+import io.papermc.paper.plugin.bootstrap.BootstrapContext;
+import io.papermc.paper.plugin.bootstrap.PluginBootstrap;
+import io.papermc.paper.plugin.bootstrap.PluginProviderContext;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public class SecureBootBootstrap implements PluginBootstrap {
```

Comment:
This class is unnecessary. It's only used by the [`paper-plugin.yml`](https://docs.papermc.io/paper/dev/getting-started/paper-plugins#bootstrapper) file, which this plugin lacks anyway.

Reply:
My original goal was to make the plugin a paper plugin, hence the class, but I kept getting weird errors so I decided to drop it for now